client/web/compose/src/components/PageBlocks/MetricConfigurator/index.vue

```diff
@@ -344,10 +334,10 @@ export default {
 
     metricFields () {
       return [
-        { value: 'count', text: 'Count' },
+        { name: 'count', label: 'Count' },
         ...this.fields.filter(f => f.kind === 'Number')
-          .map(({ name }) => ({ value: name, text: name }))
-          .sort((a, b) => a.text.localeCompare(b.text)),
+          .map(({ name, label }) => ({ name, label }))
```

Comment:
Since now you're using vue-select, you dont need to transform the fields anymore.
So no map